### PR TITLE
Add exercise 7

### PR DESCRIPTION
**Pull request by nguyenvanman**

### CODE REVIEW

File: Exercise_7/Program.cs [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            ConvertJsonToXml("bus-services-12.json", "bus-services-12.xml");
+            Console.ReadKey();
+        }
+        static void ConvertJsonToXml(string jsonInputPath, string xmlOutputPath)
+        {
+            var xmlString = XDocument.Load(JsonReaderWriterFactory.CreateJsonReader(
+                Encoding.UTF8.GetBytes(ReadFromFile(jsonInputPath)), new XmlDictionaryReaderQuotas())).ToString();
+            WriteToFile(xmlString, xmlOutputPath);
+        }
+        static string ReadFromFile(string filePath)
```
Comment (voquanghoa) [inline Exercise_7/Program.cs]: Read comments from other pull requests.

File: Exercise_7/Program.cs [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+using System.Xml;
+using System.Xml.Linq;
+
+namespace Exercise_7
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            ConvertJsonToXml("bus-services.json", "bus-services.xml");
```
Comment (voquanghoa) [inline Exercise_7/Program.cs]: Create a list and call. `files.Foreach(ConvertJsonToXml)` where `ConvertJsonToXml` is `static void ConvertJsonToXml(string fileName)` and fileName without `.json`

Comment (nguyenvanman) [reply Exercise_7/Program.cs]: Done

File: Exercise_7/Program.cs (hunk 1 above)
Comment (nguyenvanman) [reply Exercise_7/Program.cs]: Done

File: Exercise_7/Program.cs [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+using System.Xml;
+using System.Xml.Linq;
+
+namespace Exercise_7
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            List<string> files = new List<string>(){"bus-services", "bus-services-5", "bus-services-7", "bus-services-8", "bus-services-11", "bus-services-12" };
```
Comment (voquanghoa) [inline Exercise_7/Program.cs]: Enter a new line for each item.

File: Exercise_7/Program.cs [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+        }
+        static void ConvertJsonToXml(string fileName)
+        {
+            string fileInputPath = $"{fileName}.json";
+            string fileOutputPath = $"{fileName}.xml";
+            var xmlString = XDocument.Load(JsonReaderWriterFactory.CreateJsonReader(
+                Encoding.UTF8.GetBytes(ReadFromFile(fileInputPath)), new XmlDictionaryReaderQuotas())).ToString();
+            WriteToFile(xmlString, fileOutputPath);
+        }
+        static string ReadFromFile(string filePath)
```
Comment (voquanghoa) [inline Exercise_7/Program.cs]: Remove these methods `ReadFromFile`  `WriteToFile`

Comment (nguyenvanman) [reply Exercise_7/Program.cs]: Done

File: Exercise_7/Program.cs (hunk 3 above)
Comment (nguyenvanman) [reply Exercise_7/Program.cs]: Done

**Pull request merged**